numpy/lib/arraysetops.py

```diff
@@ -378,8 +378,9 @@ def setxor1d(ar1, ar2, assume_unique=False):
 #    flag = ediff1d( aux, to_end = 1, to_begin = 1 ) == 0
     flag = np.concatenate(([True], aux[1:] != aux[:-1], [True]))
 #    flag2 = ediff1d( flag ) == 0
-    flag2 = flag[1:] == flag[:-1]
-    return aux[flag2]
+#    flag2 = flag[1:] == flag[:-1]
+#    return aux[flag2]
+    return aux[np.logical_and(flag[1:], flag[:-1])]
```

Comment:
I think they are internally translated to the same calls, but I'd day it's more common to use the bitwise operators for boolean arrays, i.e. do `flag[1:] & flag[:-1]`.